r-package/R/read_immediate_region.R

```diff
@@ -1,68 +1,63 @@
-#' Download shape files of Brazil's Immediate Geographic Areas as sf objects

```

Comment:
What exactly has been changed here? Github indicates that you have replaced the whole function